force-app/main/default/lwc/errorPanel/errorPanel.js

```diff
@@ -6,24 +6,56 @@ export default class ErrorPanel extends LightningElement {
 
     @track viewDetails = false;
 
-    _errors;
+    @track _errors;
```

Comment:
I think we defined that private props that are tracked should not be prefixed (`@track errors`)?